lib/Ohmbrewer_Pump.cpp

```diff
@@ -61,6 +63,7 @@ int Ohmbrewer::Pump::doWork() {
  * @returns The time taken to run the method
  */
 int Ohmbrewer::Pump::doDisplay(Ohmbrewer::Screen *screen) {
+    screen->print("Pump");
```

Comment:
The method in Screen that displays all the pumps (Screen::displayPumps()) already adds a header that says "Pumps" and you didn't change that. This seems extraneous...